Projects/UOContent/Items/Weapons/BaseWeapon.cs

```diff
@@ -1864,7 +1864,9 @@ public virtual void OnHit(Mobile attacker, Mobile defender, double damageBonus =
                 move = null;
             }
 
-            var ignoreArmor = a is ArmorIgnore || move?.IgnoreArmor(attacker) == true;
+            WeaponAbility bladeweavingAbi;
+            var bladeweaving = Bladeweave.BladeWeaving(attacker, out bladeweavingAbi);
```

Comment:
We actually lose short circuiting by having this on a separate variable. Let's inline this to the conditional.